src/sorting/Sort.ts

```diff
@@ -61,8 +61,21 @@ export const selectionSort = <T>(items: T[]): T[] => {
 }
 
 export const insertionSort = <T>(items: T[]): T[] => {
-  throw new Error('Not Implemented!')
-  return []  
+  let output = clone(items)
+  
+  for (let i=1; i < output.length; i++){
+     let currVal = output[i], j = i - 1;
+    
+     //loop through already sorted values to the left of the currVal index
+     while(j>=0 && output[j] > currVal){
```

Comment:
The name `j` is not a good var name in this case. `i` is fine as it doesn't really do anything but move you forward through the list. But if you understand the algorithm, you realize that `j` has a well-defined purpose - it marks the position of the already sorted sublist (the items below `i`). That in mind, I would rename `j` to `position`:
```
while (position >= 0 && output[position] > currentValue) {
...
}
```